doc/man3/OPENSSL_init_crypto.pod

```diff
@@ -141,17 +141,21 @@ CAPI engine (if available). This not a default option.
 With this option the library will automatically load and initialise the
 padlock engine (if available). This not a default option.
 
-=item OPENSSL_INIT_ENGINE_DASYNC
+=item OPENSSL_INIT_ENGINE_AFALG
 
 With this option the library will automatically load and initialise the
-DASYNC engine. This not a default option.
+AFALG engine. This not a default option.
 
 =item OPENSSL_INIT_ENGINE_ALL_BUILTIN
 
 With this option the library will automatically load and initialise all the
-built in engines listed above with the exception of the openssl and dasync
+built in engines listed above with the exception of the openssl and afalg
 engines. This not a default option.
 
+=item OPENSSL_INIT_BASE_ONLY
+
+This option is for internal use.
```

Comment:
Then does it need to be in a public header? If you move it to an internal header, please make sure that we don't reuse it by adding comments.

Reply:
I don't think this define really hurts the eye and given it's name one would anyway guess
what it does.